boards/posix/doc/board.rst

```diff
@@ -0,0 +1,158 @@
+.. _simple_process:
+
+Native POSIX execution (Simpe_process)
+######################################
+
+Overview
+********
+
+Using this board, Zephyr applications can be compiled together with
+the Zephyr kernel as exectuables for the native / host OS.
+In this manner one can debug and instrument the code with better visibility
+and debug-ability. The resulting application is just a normal console
+executable.
+
```

Comment:
This is a good place to add a section documenting host system dependencies:  Does this only work on Linux hosts?  What about macOS or Windows Subsystem for Linux (or msys2)?

Also, document software and libraries: you mention installing gcc-multilib package on Ubuntu below.